content/exporters/custom-exporter/Java/Metrics.md

```diff
@@ -0,0 +1,297 @@
+  <groupId>io.opencensus</groupId>
+  <artifactId>exporter-stats-example</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>quickstart</name>
+  <url>http://maven.apache.org</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <opencensus.version>0.19.2</opencensus.version> <!-- The OpenCensus version to use -->
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-api</artifactId>
+      <version>${opencensus.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-impl</artifactId>
+      <version>${opencensus.version}</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-exporter-metrics-util</artifactId>
+      <version>${opencensus.version}</version>
+    </dependency>
+  
+  </dependencies>
+  <build>
+    <extensions>
+      <extension>
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.5.0.Final</version>
+      </extension>
+    </extensions>
+
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.7.0</version>
+          <configuration>
+            <source>1.8</source>
+            <target>1.8</target>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>appassembler-maven-plugin</artifactId>
+          <version>1.10</version>
+          <configuration>
+            <programs>
+              <program>
+                <id>ExampleStatsExporter</id>
+                <mainClass>io.opencensus.examples.exporter.stats.ExampleStatsExporter</mainClass>
+              </program>
+            </programs>
+          </configuration>
+        </plugin>
+      </plugins>
+
+    </pluginManagement>
+
+  </build>
+</project>
+{{</highlight>}}
+{{</tabs>}}
+
+Run the example with the command and observe the exported metrics output to the console
+```shell
+mvn install
+mvn exec:java -Dexec.mainClass=io.opencensus.examples.exporter.stats.ExampleStatsExporter
+...
+INFO: Timeseries to export:
+Seconds Nanos   Value
+1550971415      491000000       ValueDistribution{value=Distribution{count=100, sum=865.0, sumOfSquaredDeviations=3068.7500000000005, bucketOptions=ExplicitOptions{bucketBoundaries=[5.0, 10.0, 15.0, 20.0]}, buckets=[Bucket{count=25, exemplar=null}, Bucket{count=33, exemplar=null}, Bucket{count=24, exemplar=null}, Bucket{count=18, exemplar=null}, Bucket{count=0, exemplar=null}]}}
+```
+
+## References
+
+Name|Link
+---|---
+Exporter Utilities JavaDoc |[io.opencensus.exporter.metrics.util.*](https://www.javadoc.io/doc/io.opencensus/opencensus-exporter-metrics-util/0.19.2)
```

Comment:
nit: remove "0.19.2", then it will always point to the newest javadoc version.

Reply:
Changed as suggested in new push